Content.Server/RandomMetadata/RandomMetadataSystem.cs

```diff
@@ -47,13 +47,23 @@ public string GetRandomFromSegments(List<string> segments, string? separator)
         var outputSegments = new List<string>();
         foreach (var segment in segments)
         {
-            if (_prototype.TryIndex<DatasetPrototype>(segment, out var proto)) {
+            if (_prototype.TryIndex<LocalizedDatasetPrototype>(segment, out var localizedProto))
+            {
+                var random = _random.Pick(localizedProto.Values);
```

Comment:
isnt there a helper to pick and localize a value

Reply:
> isnt there a helper to pick and localize a value

maybe you are thinking of this? (which is whats being used here)
![image](https://github.com/space-wizards/space-station-14/assets/60196617/23b3c44e-898a-4383-a87a-c934a69aefc0)

Reply:
> Not that I'm aware of, but that would be a nice thing to have! I'll look into adding it.

![image](https://github.com/space-wizards/space-station-14/assets/60196617/faef3b53-4669-4762-b847-5c814c87358e)

My man you addded this like a week ago

Reply:
Heh. Well, that doesn't localize it... but it could, and maybe it should.

Reply:
i meant that this already handles .Values for you so you shouldnt call it like Pick(localizedProto.Values), correct?

Reply:
also it should use Next instead of le funny iterator i dont trust epic gamer language to optimise it

Reply:
Mm, the battle between optimization and DRYness. It's like two lines though, so fair enough.